Source/cursor.cpp

```diff
@@ -209,8 +209,8 @@ void CheckCursMove()
 	sy -= ScrollInfo._syoff;
 
 	if (ScrollInfo._sdir != 0) {
-		sx += ((plr[myplr]._pVar6 + plr[myplr]._pxvel) >> 8) - (plr[myplr]._pVar6 >> 8);
-		sy += ((plr[myplr]._pVar7 + plr[myplr]._pyvel) >> 8) - (plr[myplr]._pVar7 >> 8);
+		sx += ((plr[myplr]._pVar6 + plr[myplr]._pxvel) / 256) - (plr[myplr]._pVar6 / 256);

```

Comment:
It's possible that original Diablo also used `/ 256` and the shifts were the result of the optimizer

Reply:
I think they actually used shifts, we had *= 2 in some places which was bin exact on vc6 but vc5 actually only accepted <<= 1, and they liked to "optimize" things so I'd say they went with shifts :P

Reply:
Diablo used shifts for almost everything, VC6 only ompimize it on arare occasions, and VC5 probably even less so (code is 78% verified with VC5).